src/style.css

```diff
@@ -1 +1,84 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  background: #fff;
+}
+
+.header__nav {
+  display: flex;
+  padding: 0 20px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+  align-self: center;
+  justify-self: center;
+}
+
+.nav__item {
+  padding-right: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: rgba(0, 0, 0, 1);
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-style: normal;
+  text-transform: uppercase;
+}
+
+.apple {
+  color: rgba(0, 172, 220, 1);
+}
+
+.nav__link:hover {
+  color: #00acdc;
+  cursor: pointer;
+}
+
+.is-active:hover {
+  color: #00acdc;
+}
+
+.header__logo:hover {
+  color: rgba(0, 172, 220, 1);
+  border: 1px solid rgba(0, 172, 220, 1);
+}
+
+.apple::after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 57px;
+  left: 254px;
+  width: 37px;
+  height: 3px;
+  border-radius: 5px;
+  background-color: rgba(0, 172, 220, 1);
+}
```

Comment:
Why do you need a separate class when you already have `is-active`?
Something is wrong with your position 
![image](https://user-images.githubusercontent.com/57747481/173302973-02e815ec-4848-4be2-9850-791c8ba0a894.png)